### PR TITLE
JavaVersionEnum improvements

### PR DESCRIPTION
**Pull request by omirpd**
Modified method inside JavaVersionEnum class (Signature and body)
which return a JavaVersionEnum object from an majorVersion (integer)

### CODE REVIEW

File: src/br/com/codigomix/javer/util/enums/JavaVersionEnum.java [hunk 1]
```diff
@@ -6,60 +6,39 @@
...
+	}
+
+	JavaVersionEnum(Integer majorVersion, String description) {
+		this.majorVersion = majorVersion;
+		this.description = description;
+	}
+
+	public static JavaVersionEnum getByMajorVersion(Integer majorVersion) {
+
+		JavaVersionEnum javaVersionEnum = versions.get(majorVersion);
```
Comment (codigomix) [inline src/br/com/codigomix/javer/util/enums/JavaVersionEnum.java]: What about versions.getOrDefault(majorVersion, JAVA_SE_UNDEFINED) ?

File: src/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -1 +0,0 @@
-Main-Class: br.com.codigomix.javer.Main
```
Comment (codigomix) [inline src/META-INF/MANIFEST.MF:1]: We need exactly this content, with an empty line at the end of the file.
---
```
Manifest-Version: 1.0
Main-Class: br.com.codigomix.javer.Main

```

File: src/br/com/codigomix/javer/model/JaverComponent.java [hunk 3]
```diff
@@ -19,7 +19,7 @@ public JavaVersionEnum getMajorVersion(Path path) throws JaverException {
 			byte[] readBytes;
 			readBytes = Files.readAllBytes(path);
 			int major = readBytes[6] + readBytes[7];
-			return JavaVersionEnum.getJavaVersionEnum(major);
+			return JavaVersionEnum.getByMajorVersion(major);
```
Comment (codigomix) [inline src/br/com/codigomix/javer/model/JaverComponent.java:22]: good catch

**Review by codigomix — CHANGES_REQUESTED** [review]
Please review the comments.

File: src/br/com/codigomix/javer/util/enums/JavaVersionEnum.java (hunk 1 above)
Comment (omirpd) [reply src/br/com/codigomix/javer/util/enums/JavaVersionEnum.java]: Great suggestion, It'll go on the next commit/push

**Comment by omirpd** [comment]
Changes will be resent

**Pull request closed without merging**